src/App.tsx

```diff
@@ -400,68 +432,70 @@ function App() {
-                  const authUrl = `https://accounts.spotify.com/authorize?client_id=${clientId}&response_type=token&redirect_uri=${encodeURIComponent(
-                    redirectUri
-                  )}&scope=${encodeURIComponent(
-                    scopes
-                  )}&grant_type=authorization_code`;
-                  (window as any).location = authUrl;
-                }}
-              >
-                Login to Spotify
-              </button>
-            )}
-            {/* <div className="flex-grow"> */}
-            <div className="">
-              <div className="flex-none gap-2"></div>
-            </div>
-            <div className="flex flex-row text-blue-200">
-              {/* <div className="flex flex-col w-[20%] mr-10 mt-10 relative min-h-[100%]"> */}
-              {/* </div> */}
-              <div className="flex flex-col">
-                <div className="flex">
-                  <div className="form-control ml-2 mt-2">
-                    <form
-                      onSubmit={(e) => {
-                        e.preventDefault();
-                        console.log("search happening");
-                        if (apiServices) {
-                          apiServices
-                            .songSearch(searchVal)
-                            .then((res: Response) => {
-                              console.log("res", res);
-                              return res.json();
-                            })
-                            .then((res: Response) =>
-                              setSongTable((res as any).tracks.items)
-                            );
-                        }
-                      }}
-                    >
-                      <input
-                        value={searchVal}
-                        onChange={(e) => setSearchVal(e.target.value)}
-                        type="text"
-                        placeholder="Search"
-                        className="input input-bordered ml-2 mt-!"
-                      />
-                    </form>
-                  </div>
 
-                  {/* <div className="dropdown dropdown-end">
+      {!accessToken && (
+        <button
+          id="login"
+          onClick={() => {
+            const scopes =
+              "user-read-playback-state user-modify-playback-state streaming user-top-read user-read-private user-read-email user-library-modify";
+            const authUrl = `https://accounts.spotify.com/authorize?client_id=${clientId}&response_type=token&redirect_uri=${encodeURIComponent(
+              redirectUri
+            )}&scope=${encodeURIComponent(
+              scopes
+            )}&grant_type=authorization_code`;
+            (window as any).location = authUrl;
+          }}
+        >
+          Login to Spotify
+        </button>
+      )}
+      {accessToken && (
+        <div className="flex flex-col min-h-screen">
```

Comment:
componentize this (idk what it is, so a name would be helpful) and render that component instead of all this markup